CHANGELOG.md

```diff
@@ -24,7 +24,7 @@
 **Internal**:
 
 - Add new metrics on Relay's performance in dealing with buckets of metric aggregates, as well as the amount of aggregated buckets. ([#1070](https://github.com/getsentry/relay/pull/1070))
-- Add the exclusive time of a span. ([#1061](https://github.com/getsentry/relay/pull/1061))
+- Add the exclusive time of a span. ([#1061](https://github.com/getsentry/relay/pull/1061)), ([#1083](https://github.com/getsentry/relay/pull/1083))
```

Comment:
@Zylphrex the changelog for 1083 should be moved to the top at the `Unreleased` section. This line is associated with v21.9.0.